test/e2e-setup/bicep/cluster-only.bicep

```diff
@@ -0,0 +1,37 @@
+@description('If set to true, the cluster will not be deleted automatically after few days.')
+param persistTagValue bool = false
+
+@description('Name of the hypershift cluster')
+param clusterName string
```

Comment:
Why accept as input instead of reporting the name back as output.

Reply:
this is the same approach as used in other bicep files in this level - we can add a default value and report the cluster name back as an output there, but that means we will have to change the other bicep files there as well - let's consider that in a separate PR